qualysvulnerabilitymanagement_connector.py

```diff
@@ -606,16 +606,22 @@ def _handle_scan_summary(self, param):
             response = response["SCAN_SUMMARY_LIST"]
             if not isinstance(response.get("SCAN_SUMMARY"), list):
                 response["SCAN_SUMMARY"] = [response["SCAN_SUMMARY"]]
```

Comment:
Everything under this line is using response["SCAN_SUMMARY"]
wouldn't it be cleaner to set it as a separate variable?
Same with host_summary later on in for loop,  you don't need to use full 
response["SCAN_SUMMARY"][scan_idx]["HOST_SUMMARY"][host_idx] as you stated res_data, dicts are mutable.

Reply:
Applied